History.md

```diff
@@ -6,6 +6,7 @@
 
 ### Bugfixes
```

Comment:
This change was not supposed to be in this PR! I thought I had pushed this directly to master previously :(